### ISSUE TITLE
App crash while month view change to week or day 

### ISSUE THREAD

**Issue by fawadshahdev** [intljusticemission/react-big-calendar#1410]
Hello, 
I am trying to changing month view from month to day or week but in meantime I got error and app is going to crash. I passed all props like `startAccessor='start' endAccessor='end'` or `startAccessor='startDate' endAccessor='endDate'` but I don't achieve my goal . When I try to pass props `startAccessor='startDate' endAccessor='endDate'` app is not going to crash but events are not showing to it related dates. please help me thanks

**Comment by ffjanhoeck** [comment]
It would be helpful to get a reproduceable example on codesandbox or to see your code.

**Comment by ChristopherReid404** [comment]
We had crash issues switching between month/day views running v0.20.2.  We were able to fix by updating to v0.20.4.

**Comment by crashtech** [comment]
I had the same problem. In order to fix it, you have to make sure that both `start` and `end` (or `startDate` and `endDate``) are date objects, not strings., something like this:

```jsx
this.events = (result.data || []).map(item => {
  item['start']  = item['start'] && moment(item['start_at']).toDate()
  item['end'] = item['end'] && moment(item['finish_at']).toDate()
  return item
})
```
